refml/topic_modeling/refml_sql.py

```diff
@@ -1,12 +1,61 @@
 from refml.topic_modeling.preprocess import stem_and_tokenize
 from refml.topic_modeling.model import refml_nmf_tsne
 import pandas as pd
+from sqlalchemy import create_engine
+from sqlalchemy.types import Integer, String, Float
+import cx_Oracle
 
-def runrefml(input, n_topics, n_top_words, n_dimensions, perplexity):
-    df['Text_Processed'] = input.apply(stem_and_tokenize)
-    return refml_nmf_tsne(df.Text_Processed, n_topics, n_top_words, n_dimensions, perplexity)
+def hero_to_df(Userid, Password, Host, Port, SID, query):
+    dsn_tns = cx_Oracle.makedsn(host=Host, port=Port, sid=SID)
+    conn = cx_Oracle.connect(user=Userid, password=Password, dsn=dsn_tns)
 
-def refml_for_sql(sql_in, input_column, n_topics, n_top_words, n_dimensions, perplexity):
-    input = pd.from_sql(sql_in)
-    output = runrefml(input.input_column, n_topics, n_top_words, n_dimensions, perplexity)
-    return pd.to_sql(output)
\ No newline at end of file
+    return pd.read_sql_query(query, conn)
+
+def refml_to_sql(df, input_column, n_topics, n_top_words, n_dimensions, perplexity):
+    df['Text_Processed'] = df[input_column].apply(stem_and_tokenize)
+    export = refml_nmf_tsne(df.Text_Processed, n_topics, n_top_words, n_dimensions, perplexity)
+
+    # Create in memory sqlite3 database
+    engine = create_engine('sqlite://', echo=False)
+    return export.to_sql('refml_export',
```

Comment:
the schema seems odd to  me - are there only 4 columns? could we rename 0 and 1, dim_0 and dim_1 perhaps?  is there no `ref_id` or something that we could use to associate each row to a reference?